LICENSE

```diff
@@ -1,11 +1,14 @@
 ISC License
 
-Copyright (c) 2020, Lucide Contributors
+Copyright (c) 2022, Lucide Contributors
 
 Permission to use, copy, modify, and/or distribute this software for any
 purpose with or without fee is hereby granted, provided that the above
 copyright notice and this permission notice appear in all copies.
 
+Copyright for portions of project Lucide are held under the MIT license
```

Comment:
I have no idea about how to deal with this kind of stuff but I would put this at the top.